op.c

```diff
@@ -9646,6 +9646,35 @@ Perl_newWHILEOP(pTHX_ I32 flags, I32 debuggable, LOOP *loop,
     return o;
 }
 
+#define resolve_lex_sv(t) S_resolve_lex_sv(aTHX_ t)
+static SV *
+S_resolve_lex_sv(pTHX_ PADOFFSET t) {
```

Comment:
This seems remarkably similar to find_lexical_cv() (which handles some cases this doesn't)

Reply:
Ah, thank you. I knew this functionality had to be there somewhere, but I couldn't find anything in perlapi/perlintern and I didn't spot it in the code.